packages/components/src/schema/props/spell-check.ts

```diff
@@ -0,0 +1,20 @@
+import type { Generic } from 'adopted-style-sheets';
+
+import { watchBoolean } from '../utils';
+
+/* types */
+export type SpellCheckPropType = boolean;
+
+/**
+ * Defines whether the spellcheck should be enabled.
```

Comment:
```suggestion
 * Defines whether the browser should check the spelling and grammar.
```
Vorschlag: Selben Kommentar wie in den Komponenten verwenden.